### PR TITLE
Checkout standalone

### PR DESCRIPTION
**Pull request by krico**
widget now creates anonymous shopping cart and hands over logic to checkout-module in a popup window.

### CODE REVIEW

File: schedule/schedule-appengine/src/main/client/checkout-window.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <link href="/build/css/dep-main.css" rel="stylesheet">
     <link href="/build/css/booking.css" rel="stylesheet">
 </head>
-<body class="checkout-window">
+<body style="height:2000px" ng-controller="CheckoutWindowPageController as pageVm">
```
Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/checkout-window.html]: Is it safe to assume this height?

File: schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js [hunk 2]
```diff
@@ -218,31 +218,93 @@
...
-                    $location.path('/checkout');
-                }, function () {
-                    // TODO
+        function confirmPopup(r) {
+            jasDialogs.ok('Proceed in new window', 'Booking is done via Jasify. ' +
+                'A new window will be opened where you will be able to Sign In or Create an account and ' +
+                'proceed to your checkout.',
+                onOk);
+            function onOk() {
+                var w = $window.innerWidth || 820;
```
Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js]: Again are these dimensions safe?

File: schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                    <div ng-controller="SignInController as vm"
+                         ng-include="'authenticate/sign-in.html'"></div>
+                </div>
+            </div>
+        </div>
+        <div class="col-md-6">
+            <div class="card">
+                <div class="card-header">
+                    <h2>Create Account
+                        <small>If you don't have Jasify account, create it</small>
```
Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html]: do not have a Jasify account

File: schedule/schedule-appengine/src/main/client/checkout-window.html (hunk 1 above)
Comment (krico) [inline schedule/schedule-appengine/src/main/client/checkout-window.html]: Not really, it's a hack...  I raised a "request for help" here #557

File: schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js (hunk 2 above)
Comment (krico) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/booking-via-jasify.js]: I popup a window based on the size of the current window `$window.innerWidth`.  The 820 is just a fallback.  This one I think is safe.

File: schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html (hunk 3 above)
Comment (krico) [inline schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html]: yep #558

Comment (pauljanes) [inline schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html]: Have a Jasify account

Without the "a" it sounds like a foreigner wrote it

**Pull request merged**